workflow/controller/operator.go

```diff
@@ -3096,6 +3097,20 @@ func (woc *wfOperationCtx) retryStrategy(tmpl *wfv1.Template) *wfv1.RetryStrateg
 	return woc.execWf.Spec.RetryStrategy
 }
 
+func (woc *wfOperationCtx) cache(tmpl *wfv1.Template) *wfv1.Cache {
```

Comment:
This function name should be more descriptive. `getCache` would be better.

Reply:
Not sure I'm a fan of this function returning a field from a parameter or a receiver and I think it's largely unnecessary. It's only used in one place.

I would linline it instead.

Reply:
Sure. Will update.